stan/math/prim/mat/fun/cov_exp_quad.hpp

```diff
@@ -84,8 +86,8 @@ namespace stan {
                   Eigen::Dynamic, Eigen::Dynamic>
     cov_exp_quad(const std::vector<T_x1>& x1,
                  const std::vector<T_x2>& x2,
-                 T_sigma& sigma,
-                 T_l& l) {
+                 const T_sigma& sigma,
+                 const T_l& l) {
       using std::exp;
```

Comment:
nvm, I see it handles int arguments. Fixed prim to include `using std::exp` for both functions, changed calls to `exp` in rev to explicit `std::exp` because they're always acting on doubles.